app/src/docs/content/core/introToCore.mdx

```diff
@@ -84,9 +84,9 @@ All of the main resources types created and used within Core can be shared with
   figCaption="Share button in canvas editor"
 />
 
-When you share a resource, you can share it either privately or publicly. A private share means the link is only accessible to the users whose email address it has been shared with. A public share means anyone with the link — and importantly, anyone using the public API — can access your resource. 
+When you share a resource, you can share it either privately or publicly. A private share means the link is only accessible to the users whose Ethereum address it has been shared with. A public share means anyone with the link — and importantly, anyone using the public API — can access your resource. 
```

Comment:
`...you can either share it privately or publicly` (which placement of "either is correct?)

Reply:
Your grammar tends to be better than mine but I think the original sounds slightly better

Reply:
Sure, I'll trust your native ear